### DISCUSSION TITLE
Unable to use Proxy in act - Containers

### DISCUSSION THREAD

**Discussion by Vaitheesh-SS** [nektos/act#1570] [Q&A]
Hello!,

I'm trying to run a local test actions on my VM using nektos/act and I'm facing issues in reaching internet especially `Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease Could not resolve 'archive.ubuntu.com` through my Proxy configured on my Ubuntu VM. The docker containers spin from act is unable to inject the Proxy Env's which I defined in `~/.docker/config.json` but this works well when creating containers using docker cli

`HTTPS_PROXY=http://localhost:3128
HTTP_PROXY=http://localhost:3128` -- also tried with VM IP

Tried with `act -j deploy --env HTTP_PROXY=http://localhost:3128/ --env HTTPS_PROXY=http://localhost:3128/` - no luck !

**Comment by sysid** [comment]
For me on mac this is working:
`act --env https_proxy=http://host.docker.internal:3128 --env http_proxy=http://host.docker.internal:3128`
